Changelog.md

```diff
@@ -1,3 +1,15 @@
+## 0.2.6 - 2024-04-12
+
+### Changed
+
+-   Updated Nordic Semiconductor documentation links.
+
+### Fixed
+
+-   A skip button is now shown in the verification step if verification fails.
```

Comment:
Shouldn't this be a ### Changed or ### Added?

Reply:
There was a skip button before but it was shown during weird times only.
I get why my wording would give that impression though.

Reply:
Add "again" between "now" and "shown"?